browser/js/global.js

```diff
@@ -303,8 +303,11 @@ class InitializerBase {
 			brandingLink.setAttribute("href", this.brandingUriPrefix + theme_prefix + 'branding-desktop.css');
 		}
 
-		document.getElementsByTagName("head")[[0]].appendChild(link);
-		document.getElementsByTagName("head")[[0]].appendChild(brandingLink);
+		const otherStylesheets = document.querySelectorAll('link[rel="stylesheet"]');
+		const lastOtherStylesheet = otherStylesheets[otherStylesheets.length - 1];
+
+		lastOtherStylesheet.insertAdjacentElement('afterend', brandingLink);
+		lastOtherStylesheet.insertAdjacentElement('afterend', link);
```

Comment:
don't we want branding at the very end?

Reply:
that's correct, I think perhaps I should change those statements to make it more obvious what was going on - while I was writing it I was iffy, now I'm even less sure it's readable

To explain...:
```
lastOtherStylesheet
```
(branding stylesheet gets inserted)
```
lastOtherStylesheet
brandingStylesheet
```
(device specific stylesheet gets inserted)
```
lastOtherStylesheet
deviceSpecificStylesheet
brandingStylesheet
```

notice how, by inserting after the child, we have flipped the stylesheets

...

contrast this with if we were inserting inside <head>, where we would always be inserting at the end, and you'll be able to see why the branding/device specific lines were flipped here

Reply:
I've rewritten that to be clearer